src/Components/BankDebitForm/BankDebitForm.tsx

```diff
@@ -35,9 +37,11 @@ export const BankDebitForm: FC<Props> = ({
   const [isSaveAccountChecked, setIsSaveAccountChecked] = useState(true)
   const tracking = useTracking()
 
-  const trackPaymentElementEvent = event => {
+  const onPaymentElementChange = event => {
     const trackedEvents: any[] = []
```

Comment:
PS: I don't think we need to create an array and push events to it, and then loop over it to track. If anyone has no objection, I'll remove it here: https://github.com/artsy/force/pull/10703